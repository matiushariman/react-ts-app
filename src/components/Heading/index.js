import styled from 'styled-components';

export const Heading = styled.h1`
  font-size: 16px;
  font-weight: bold;
  text-transform: uppercase;
`;

export default Heading;
