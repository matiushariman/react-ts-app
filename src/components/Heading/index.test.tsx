import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';

import Heading from '.';

test('renders the component', () => {
  const { container } = render(<Heading />);

  expect(container).toMatchSnapshot();
});
