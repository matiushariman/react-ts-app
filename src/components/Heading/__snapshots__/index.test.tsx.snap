// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
.c0 {
  font-size: 16px;
  font-weight: bold;
  text-transform: uppercase;
}

<div>
  <h1
    class="c0"
  />
</div>
`;
